cli/src/index.ts

```diff
@@ -96,6 +102,11 @@ async function main() {
 				console.log(`\t-bf make|bob|imd|json\tCreate build files of a specific format`);
 				console.log(`\t\t\tExample: -bf make`);
 				console.log(``);
+				console.log(`\t-bl <name>\tSet the BIN_LIB environment variable`);
+				console.log(`\t\t\tbased on a user provided branch na,e`);
+				console.log(`\t\t\tExample: -bl feature/123-cool-idea`);
```

Comment:
so in these examples, what does the library name become?

Reply:
For `feature/123-cool-idea` it would be `FEA123`